packages/core/src/button/Button.tsx

```diff
@@ -72,15 +89,41 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
           withBaseName(variant),
           {
             [withBaseName("disabled")]: disabled,
+            [withBaseName(`loading-${variant}`)]: loading,
             [withBaseName("active")]: active,
           },
           className
         )}
+        aria-disabled={disabled}
```

Comment:
aria-disabled is handled in useButton, this overrides it?

Reply:
removed